.github/workflows/pr.yml

```diff
@@ -37,12 +37,13 @@ jobs:
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
-      - name: Install Doxygen
-        run: |
-          sudo apt-get update -q
-          sudo apt-get install -y doxygen
-      - name: Check Doxygen docs
+      - name: Install and Check Doxygen
         run: |
+          conda init bash
```

Comment:
Let’s move all of this to a standalone bash script.

Reply:
Model it after https://github.com/rapidsai/cudf/blob/branch-24.12/ci/check_style.sh.

Reply:
Do we need conda after all? We could just wget one of the binary releases from the doxygen GH repo. I'm not saying I'm against conda, I'm just wondering if it comes with any additional overhead compared to downloading the raw binary and placing it in an executable path.